main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/giantswarm/micrologger"
+)
+
+const (
+	// Port which listens the server
+	Port = "80"
+)
+
+func main() {
+	var lHelp bool
+	var sHelp bool
+	flag.BoolVar(&lHelp, "help", false, "Print help usage.")
+	flag.BoolVar(&sHelp, "h", false, "Print help usage.")
+	flag.Parse()
+	if lHelp || sHelp {
+		flag.Usage()
+		return
+	}
+
+	ctx := context.Background()
+	mux := http.NewServeMux()
+	logger, err := micrologger.New(micrologger.Config{})
+	if err != nil {
+		panic(err)
+	}
+
+	mux.Handle("/healthz", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(""))
+	}))
```

Comment:
What kind of liveness are we actually trying to verify here? Looks a bit similar to https://github.com/giantswarm/e2e-app. Where is the difference though?

Reply:
yeah it was my inspiration. The idea is to enable istio ingress as main IC on KVM envs. 
For nginx deployments, there is a `/healthz` endpoint returning `200`. In istio we need to configure an app (and a virtual service and gateway) to verify it. I could use it the e2e app but it has different purpose and we can have conflicts. For example, liveness cannot return content in that endpoint or prometheus will fail to parse the response payload.